ui/pages/asset/util.ts

```diff
@@ -64,3 +64,31 @@ export const chainSupportsPricing = (chainId: `0x${string}`) =>
 
 /** The opacity components should set during transition */
 export const loadingOpacity = 0.2;
+
+// TODO: Add unit tests
```

Comment:
Once a few of these are in I'll approve!

Reply:
Added unit test here: https://github.com/MetaMask/metamask-extension/pull/28429/commits/1a1737a7a6f215464bc6c70c163a039e972a7a0a

Reply:
Updated with bugfix here: https://github.com/MetaMask/metamask-extension/pull/28429/commits/41adb561acf19278ba2a2f52fafa678d98f8e7d4